### ISSUE TITLE
Investigate if changing storage method is necessary

### ISSUE THREAD

**Issue by Janaka-Steph** [tdex-network/tdex-app#552]
We are currently using Capacitor v3 @capacitor/storage (https://capacitorjs.com/docs/v3/apis/storage).
On Capacitor v4 it changed for this plugin https://capacitorjs.com/docs/apis/preferences.
But there are basically the same, using [UserDefaults](https://developer.apple.com/documentation/foundation/userdefaults) on iOS and [SharedPreferences](https://developer.android.com/reference/android/content/SharedPreferences) on Android. 

Even though there is no size limit on iOS UserDefaults, the doc says: "This API is not meant to be used as a local database. If your app stores a lot of data, has high read/write load, or requires complex querying, we recommend taking a look at a SQLite-based solution."

We may gain some perf using https://github.com/capacitor-community/sqlite

**Comment by tiero** [comment]
What if we use use Ionic Storage with SQlite as "storage"? This way we retain the Key/Value interface we already use 


https://github.com/ionic-team/ionic-storage#sqlite-installation

**Comment by Janaka-Steph** [comment]
Yes it is an option too. Would be this plugin https://github.com/storesafe/cordova-sqlite-storage
Last commit July 2021. We can give it a try.
